File: Iniciante/1050 - DDD/Python/1050.py
def main():
    code = int(input())

    ddds = create_map_ddd()

    city = identify_city(code, ddds)

    print(city)

def create_map_ddd():
    ddds = {
        61: "Brasilia",
        71: "Salvador",
        11: "Sao Paulo",
        21: "Rio de Janeiro",
        32: "Juiz de Fora",
        19: "Campinas",
        27: "Vitoria",
        31: "Belo Horizonte"
    }
    return ddds


def identify_city(ddd, ddds):
    if ddd in ddds:
        return ddds[ddd]
    else:
        return "DDD nao cadastrado"


if __name__ == "__main__":
    main()

File: Iniciante/1050 - DDD/TypeScript/1050.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8").trim();
  const ddd: number = Number(input);
  const ddds = createMapDDD();
  const city = identifyCity(ddd, ddds);

  console.log(city);
}

function createMapDDD(): Map<number, string> {
  const ddds = new Map<number, string>();

  ddds.set(61, "Brasilia");
  ddds.set(71, "Salvador");
  ddds.set(11, "Sao Paulo");
  ddds.set(21, "Rio de Janeiro");
  ddds.set(32, "Juiz de Fora");
  ddds.set(19, "Campinas");
  ddds.set(27, "Vitoria");
  ddds.set(31, "Belo Horizonte");

  return ddds;
}

function identifyCity(ddd: number, ddds: Map<number, string>): string {
  if (ddds.has(ddd)) {
    return ddds.get(ddd)!;
  } else {
    return "DDD nao cadastrado";
  }
}

main();
